services/azure-devops/azure-devops-helpers.js

```diff
@@ -23,15 +22,4 @@ async function fetch(serviceInstance, { url, qs = {}, errorMessages }) {
   return { status }
 }
 
-function getHeaders() {
-  const headers = {}
-  if (serverSecrets.azure_devops_token) {
-    const pat = serverSecrets.azure_devops_token
-    const auth = Buffer.from(`:${pat}`).toString('base64')
-    headers.Authorization = `basic ${auth}`
-  }
-
-  return headers
-}
```

Comment:
🎉 🍾 The `auth()` approach feels so much simpler/cleaner in these service classes, glad to see the removal of this

Reply:
Thanks! This version came out really well. I'm glad for the back-and-forth we had on #3410; think it helped a lot in getting to this point.